Tigran Kirakosov/style.css

```diff
@@ -0,0 +1,122 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+/* Highier level containers states */
+.weather-container_main {
+  padding: 80px 110px;
+  width: 800px;
+  height: 600px;
+  margin: auto;
+  font-family: 'Dosis', sans-serif;
+  font-weight: normal;
+  background-image: url("/assets/images/bg/Tokyo-bg.png");
+  border-radius: 10px;
+  color: white;
+  cursor: default;

```

Comment:
just out of curiosity - isn't `cursor: default` for `<article>`?

Reply:
Yes, I did it in order to prevent cursor change on text hovering. I feel like its better for a user not to think about widget like its something common to the rest of the environment, but instead something different. I may be wrong, but for me, it feels right

Reply:
Ok, actually I don't have any concerns about that :+1: